kpops/components/base_components/pipeline_component.py

```diff
@@ -30,6 +32,9 @@ class PipelineComponent(BaseDefaultsComponent):
         default="${pipeline_name}-",
         description="Pipeline prefix that will prefix every component name. If you wish to not have any prefix you can specify an empty string.",
     )
+    schema_type: Literal["pipeline-component"] = Field(  # type: ignore[assignment]
+        default="pipeline-component", exclude=True
+    )
```

Comment:
please move `schema_type` directly underneath the `type` attribute as it is the case for all the other component classes